docs/src/pages/guides/cli-options.md

```diff
@@ -48,6 +48,16 @@ Use the `-o` output flag to specify the output file.
 monika --create-config --postman mypostman.json -o new-monika.json
 ```
```

Comment:
since we're mentioning force in the section below, maybe better to mention that you can use force here too in create-config